account_payment_partner/wizard/payment_order_create.py

```diff
@@ -28,10 +28,10 @@ class PaymentOrderCreate(models.TransientModel):
 
     @api.model
     def extend_payment_order_domain(self, payment_order, domain):
-        res = super(PaymentOrderCreate, self).extend_payment_order_domain(
+        super(PaymentOrderCreate, self).extend_payment_order_domain(
             payment_order, domain)
         domain += ['|', '|',
                    ('invoice', '=', False),
                    ('invoice.payment_mode_id', '=', False),
                    ('invoice.payment_mode_id', '=', payment_order.mode.id)]
-        return res
+        return True
```

Comment:
Thanks for the reminder about XMLRPC. I had forgotten that one. I agree returning the result of super is a good habit to have.